proposals/NNNN-resource-representations.md

```diff
@@ -0,0 +1,119 @@
+<!-- {% raw %} -->
+
+# Resource Representations in Clang and LLVM
+
+* Proposal: [NNNN](NNNN-resource-representations.md)
+* Author(s): [Justin Bogner](https://github.com/bogner)
+* Sponsor: [Justin Bogner](https://github.com/bogner)
+* Status: **Under Consideration**
```

Comment:
Under review? Accepted?

Reply:
That makes sense. I'll bump it to accepted when I give it a number and merge.